h2o-automl/src/main/java/water/automl/api/schemas3/AutoMLV99.java

```diff
@@ -67,42 +67,42 @@ public AutoMLKeyV3(Key<AutoML> key) {
 
     if (null == autoML) return this;
 
-    this.project_name = autoML.projectName();
+    project_name = autoML.projectName();
 
     if (null != autoML._key) {
-      this.automl_id = new AutoMLKeyV3(autoML._key);
+      automl_id = new AutoMLKeyV3(autoML._key);
     }
 
     if (null != autoML.getTrainingFrame()) {
-      this.training_frame = new KeyV3.FrameKeyV3(autoML.getTrainingFrame()._key);
+      training_frame = new KeyV3.FrameKeyV3(autoML.getTrainingFrame()._key);
     }
 
     if (null != autoML.getValidationFrame()) {
-      this.validation_frame = new KeyV3.FrameKeyV3(autoML.getValidationFrame()._key);
+      validation_frame = new KeyV3.FrameKeyV3(autoML.getValidationFrame()._key);
     }
 
     if (null != autoML.getBlendingFrame()) {
-      this.blending_frame = new KeyV3.FrameKeyV3(autoML.getBlendingFrame()._key);
+      blending_frame = new KeyV3.FrameKeyV3(autoML.getBlendingFrame()._key);
     }
 
     if (null != autoML.getLeaderboardFrame()) {
-      this.leaderboard_frame = new KeyV3.FrameKeyV3(autoML.getLeaderboardFrame()._key);
+      leaderboard_frame = new KeyV3.FrameKeyV3(autoML.getLeaderboardFrame()._key);
     }
 
     // NOTE: don't return nulls; return an empty leaderboard/eventLog, to ease life for the client
     EventLog eventLog = autoML.eventLog();
     if (null == eventLog) {
       eventLog = new EventLog(autoML._key);
     }
-    this.event_log = new EventLogV99().fillFromImpl(eventLog);
-    this.event_log_table = new TwoDimTableV3().fillFromImpl(eventLog.toTwoDimTable());
+    event_log = new EventLogV99().fillFromImpl(eventLog);
+    event_log_table = event_log.table;  // for backwards compatibility
```

Comment:
@seb-h2o does it mean that `event_log_table ` is is deprecated?

Reply:
not yet, I don't want to change the clients yet.